README.md

```diff
@@ -20,14 +20,30 @@ was originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/
 
 ## Install
 
-To install this simple rubygem you can check your package manager if it is
-available for you, alternativly you can just install the ruby gem if you have a
+To install this simple ruby gem you can check your package manager if it is
+available for you, alternatively you can just install the ruby gem if you have a
 running ruby stack on your machine with this command:
 
 ```
 gem install crowbar-client
 ```
 
+## Configuration
+
+`crowbar-client` can be configured by placing a `.crowbarrc` file in your `$HOME` directory.
+The default configuration looks like:
+```
```

Comment:
Or even better just link to https://github.com/crowbar/crowbar-client/blob/master/config/crowbarrc